include/librepr/reflection/detail/arity.h

```diff
@@ -2,19 +2,67 @@
 #include <type_traits>
 
 namespace librepr::detail {
+#include <type_traits>
+
 struct Universal {
-  template <typename T>
-  [[implicit]] operator T() {}
+    template <typename T>
+    constexpr operator T();
```

Comment:
please add `[[implicit]]` here. Doesn't do anything (yet), but maybe I'll share the clang-tidy patch some day or another.